packages/rocketchat-lib/server/functions/saveUser.js

```diff
@@ -130,6 +135,10 @@ RocketChat.saveUser = function(userId, userData) {
 			updateUser.$set.name = userData.name;
 		}
 
+		if (userData.role) {
+			updateUser.$set.roles = [ (userData.role || 'user') ];
+		}
+
```

Comment:
Are we expecting to override or to add the new role?
And if we want to add more than one role?

@RocketChat/core thoughts?